sdk/communication/Azure.Communication.Sms/tests/samples/Sample1_SmsClient.cs

```diff
@@ -4,15 +4,77 @@
 using System;
 using Azure.Core;
 using Azure.Identity;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
 
 namespace Azure.Communication.Sms.Tests.samples
 {
     /// <summary>
-    /// Samples that are used in the README.md file.
+    /// Samples to send sms
     /// </summary>
     public partial class Sample1_SmsClient
     {
-        public SmsClient CreateSmsClient()
+        [Test]
+        [AsyncOnly]
+        public async Task SendSingleAndGroupSmsAsync()
+        {
+            SmsClient smsClient = CreateSmsClient();
+
+            // SendSmsOptions is an optional field. It can be used
```

Comment:
We should probably add some minimal samples too (1:1, no delivery, not tag, just basic).

Reply:
Updated single send sms scenario with no options. Or do you want me to add another test with no sms options for single and group sms both?

Reply:
Adding new samples probably is better as these samples will end up showing up in the documentation